libxrdp/xrdp_caps.c

```diff
@@ -915,6 +936,18 @@ xrdp_caps_send_demand_active(struct xrdp_rdp *self)
     out_uint16_le(s, 8);
     out_uint32_le(s, 2); /* 2 frames in flight */
 
+    /* surface commands */
+    if (self->client_info.use_fast_path & 1) /* surface commands requires fastpath output */
```

Comment:
We need defines here: 1 to enable fastpath output, 2 to enable fastpath input. The code would show *what* is being done, comment would explain *why*. Also, *commands require* or *command requires*.